### PR TITLE
 fix jindoruntime support multi-dir tierstore

### PR DESCRIPTION
**Pull request by frankleaf**
### Ⅰ. Describe what this PR does
#428 

### Ⅳ. Describe how to verify it
For verification, here's some samples:

Sample
```
apiVersion: data.fluid.io/v1alpha1
kind: Dataset
metadata:
  name: hbase
spec:
  mounts:
    - mountPoint: https://mirror.bit.edu.cn/apache/hbase/stable/
      name: hbase
---
apiVersion: data.fluid.io/v1alpha1
kind: JindoRuntime
metadata:
  name: hbase
spec:
  replicas: 2
  tieredstore:
    levels:
      - mediumtype: SSD
        path: /mnt/cache/cache1/,/mnt/cache/cache2
        quota: 1Gi,1Gi
        high: "0.9"
        low: "0.7"
```
### Ⅴ. Special notes for reviews

### CODE REVIEW

File: pkg/ddc/jindo/shutdown.go [hunk 1]
```diff
@@ -105,11 +105,41 @@ func (e *JindoEngine) destroyWorkers(workers int32) (err error) {
...
+
+	currentWorkers = int32(len(nodeList.Items))
+	if expectedWorkers >= currentWorkers {
+		e.Log.Info("No need to scale in. Skip.")
+		return currentWorkers, nil
+	}
+
+	var nodes []corev1.Node
+	if expectedWorkers >= 0 {
+		e.Log.V(1).Info("Scale in Jindo workers", "expectedWorkers", expectedWorkers)
```
Comment (cheyang) [inline pkg/ddc/jindo/shutdown.go:119]: you can remove V(1)

**Review by cheyang — CHANGES_REQUESTED** [review]

File: pkg/ddc/jindo/shutdown.go [hunk 2]
```diff
@@ -105,11 +105,41 @@ func (e *JindoEngine) destroyWorkers(workers int32) (err error) {
...
+		}
+
+		fuseGlobal, _ := runtimeInfo.GetFuseDeployMode()
+		nodes, err = e.sortNodesToShutdown(nodeList.Items, fuseGlobal)
+		if err != nil {
+			return currentWorkers, err
+		}
+
+	} else {
+		// Destroy all workers. This is a subprocess during deletion of AlluxioRuntime
```
Comment (yangyuliufeng) [inline pkg/ddc/jindo/shutdown.go:134]: Jindo

File: pkg/ddc/jindo/shutdown.go [hunk 3]
```diff
@@ -129,11 +159,38 @@ func (e *JindoEngine) destroyWorkers(workers int32) (err error) {
...
+			return nil, err
+		}
+
+		for _, node := range candidateNodes {
+			if _, found := pvcMountNodes[node.Name]; !found {
+				nodes = append(nodes, node)
+			}
+		}
+	} else {
+		// If fuses are deployed in global mode. Scaling in workers has nothing to do with fuses.
```
Comment (yangyuliufeng) [inline pkg/ddc/jindo/shutdown.go:190]: Even if scaling in workers has nothing to do with fuses, it's better to scale workers without PVC mount Pods first.

**Review by yangyuliufeng — APPROVED** [review]

File: pkg/ddc/jindo/shutdown.go (hunk 1 above)
Comment (frankleaf) [reply pkg/ddc/jindo/shutdown.go:119]: done

File: pkg/ddc/jindo/shutdown.go (hunk 2 above)
Comment (frankleaf) [reply pkg/ddc/jindo/shutdown.go:134]: done

File: pkg/ddc/jindo/shutdown.go (hunk 3 above)
Comment (frankleaf) [reply pkg/ddc/jindo/shutdown.go:190]: jindoruntime dont't supoort fuse global option, so that's will implentment and test by next time

**Comment by cheyang** [comment]
@frankleaf Please fix conflict, thank.

**Comment by frankleaf** [comment]
> @frankleaf Please fix conflict, thank.

already done

**Pull request merged**